docs/io/output/rpacket_tracking.ipynb

```diff
@@ -43,12 +43,7 @@
    "id": "4b0de6ca",
```

Comment:
<p>Instead of using bullet, it's better to use next level heading here (<code>####</code> I guess?)</p><p> </p><p>And <code>packet {10}</code> isn't super clear. Either say "10th packet" or "packet with index=10"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>